### DISCUSSION TITLE
Does the milvus-1.1.0-GPU have log4j issue?

### DISCUSSION THREAD

**Discussion by Chisdo** [milvus-io/milvus#14679] [Q&A and General discussion] [answered]
I didn't find any pom.xml in the milvus-1.1.0-GPU Docker image. Just for double-check, I'm wondering that does the milvus-1.1.0-GPU have log4j issue?

**Comment by yhmo** [comment, marked as answer]
The milvus 1.1 server is written by C++, not Java, so the server doesn't have this issue.
But the Milvus Java SDK use the log4j, since the SDK is used on client-side, the risk is low. We have upgraded the Java SDK by latest log4j component, in this commit: https://github.com/milvus-io/milvus-sdk-java/pull/237/files, the new Java SDK version is 1.1.1.

**Reply by Chisdo** [reply]
OK, thx a lot.
